datacenter/ucp/2.2/guides/admin/configure/external-auth/index.md

```diff
@@ -30,8 +30,15 @@ Now configure your LDAP directory integration.
 
 ## Default Role For All Private Collections
 
-Click the dropdown to select the permission level assigned by default to
-the private collections of new users.
+Use this setting to change the default permissions of new users. UCP applies
+the default role automatically to user accounts that are imported from an LDAP
+server.
+
+Click the dropdown to select the permission level that UCP assigns by default
```

Comment:
This sentence basically repeats what you already said before. Maybe kill be first one since you give a cool example in this one?